dockerfiles/alpine/is/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # ------------------------------------------------------------------------
 #
-# Copyright 2018 WSO2, Inc. (http://wso2.com)
+# Copyright 2021 WSO2, Inc. (http://wso2.com)
```

Comment:
No need to update the year. If it is a new file, then we need to add the licence header